projects/observability/src/shared/components/card-list/card-list.component.scss

```diff
@@ -33,21 +34,13 @@
       }
 
       &:hover {
-        background: $blue-2;
```

Comment:
We don't want to show a darker blue when user hover on selected card?

Reply:
Need to remove these on hover on card
1. border
2. background color change

To maintain consistency on hover elements, selected and un-selected cards should follow the same behavior.
I can revert this change if required.